<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowInformation</class>
 <widget class="QMainWindow" name="ShowInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_Close">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="minimumSize">
           <size>
            <width>165</width>
            <height>80</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_showinformation_top">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>150</width>
             <height>28</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Show Information</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_Showinformation_top_image">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>0</y>
             <width>48</width>
             <height>48</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_seperator">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Helvetica CE 35 Thin</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Look For : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_PersonType">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>28</height>
         </size>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Student</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Teacher</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Staff</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_ID">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_PhoneNumber">
          <property name="text">
           <string>PhoneNumber</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_ChooseLineEdit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Helvetica CE 35 Thin</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Enter ID  :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_PhoneSearch">
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="inputMask">
         <string>00000000000</string>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Phone Number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_IDSerch">
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="inputMask">
         <string>000000000</string>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_refresh">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Reload Information</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame_PersonalInfo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>240</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_fname">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>First Name : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_lname">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Last Name : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_id">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>ID : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_phone">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Phone : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_email">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Email : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_sex">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Sex : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_birthdate">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Birth Date : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_username">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Username : </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QFrame" name="frame_student">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_student_fieldname">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Helvetica CE 35 Thin</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Field Name : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_student_entryyear">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Helvetica CE 35 Thin</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Entry Year : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_student_totalterms">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Helvetica CE 35 Thin</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Total Terms : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_student_currentTerm">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Helvetica CE 35 Thin</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Current Term : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_student_totalUnits">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Helvetica CE 35 Thin</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Total Units : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_student_passedunits">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Helvetica CE 35 Thin</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Passed Units : </string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_teacher">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_teachers_degree">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Helvetica CE 35 Thin</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Degree : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_teachers_sci_board">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Helvetica CE 35 Thin</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Scientifc Board : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_teachers_field">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Helvetica CE 35 Thin</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Field : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_teachers_hoursInWeek">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Helvetica CE 35 Thin</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Hours in Week : </string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_staff">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_Staff_department">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Helvetica CE 35 Thin</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Department : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_Staff_type">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Helvetica CE 35 Thin</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Type : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_Staff_workshift">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Helvetica CE 35 Thin</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Workshift : </string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
